Makefile

```diff
@@ -18,15 +18,19 @@ endif
 
 BINDIR=$(PWD)/build
 
-.PHONY: all clean deps pack rlec_runpath_fix
+.PHONY: all build clean deps pack rlec_runpath_fix test
 
-all:
+all: build
+
+build:
 ifeq ($(wildcard build/.),)
 	mkdir -p build
 	cd build; \
 	cmake -DDEPS_PATH=../deps/install ..
 endif
 	$(MAKE) -C build
+	@echo Fixing RLEC RUNPATH...
+	@patchelf --set-rpath '$$ORIGIN:$(REDIS_ENT_LIB_PATH)' $(BINDIR)/redisai.so
```

Comment:
This fails on macOS. We should condition this with `ifeq ($(UNAME_S),Linux)` or equivalent.

Reply:
True!
I will exclude OSX at the moment, and cover that later.